src/elin_bjoernson_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from random import randint
+
+__author__ = 'Elin Bjoernson'
+__email__ = 'elinbj@nmbu.no'
+
+
+def guess_number():
+    number = 0
+    while number < 1:
+        number = int(input('Your guess: '))
+    return number
+
+
+def random_number():
+    return randint(1, 6) + randint(1, 6)
+
+
+def same_value(f, g):
```

Comment:
`f` and `g` have no meanings to be the names of arguments. Can you think of something better?